proto/redpanda/api/dataplane/v1alpha1/kafka_connect.proto

```diff
@@ -422,6 +562,13 @@ service KafkaConnectService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
       summary: "Pause connector"
       description: "Pause the connector and its tasks, which stops message processing until the connector is resumed. This call asynchronous."
+      responses: {
+        key: "202"
+        value: {
+          description: "Accepted"
```

Comment:
Could we make this a bit more descriptive like "Pause request accepted"? Also, how does the user know when the connector is actually paused?